test/build.spec.ts

```diff
@@ -292,13 +292,13 @@ describe('Invalid build input', () => {
 			.then(() => {
 				throw new Error('Error not thrown on null buildStream input');
 			})
-			.catch(BuildProcessError, () => {
-				// This is what we want
-			})
 			.catch(e => {
-				throw new Error(
-					'Incorrect error thrown on null buildStream input: ' + e,
-				);
+				// This is what we want
+				if (!(e instanceof BuildProcessError)) {
+					throw new Error(
+						'Incorrect error thrown on null buildStream input: ' + e,
+					);
+				}
 			});
 	});
 });
```

Comment:
Although the tests are passing by the CI, they fail on my laptop with Docker Desktop for macOS:
```
  1) Resolved project building
       should correctly build a resolved project for a different platform:

      AssertionError: expected 'amd64' to equal '386'
      + expected - actual

      -amd64
      +386

      at /Users/paulo/balena/balena-io-modules/resin-multibuild/build/test/build.spec.js:241:27
```
```
$ docker version
...
Server: Docker Engine - Community
 Engine:
  Version:          19.03.13
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       4484c46d9d
  Built:            Wed Sep 16 17:07:04 2020
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          v1.3.7
  GitCommit:        8fba4e9a7d01810a393d5d25a3621dc101981175
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```
Trying to figure out why...